.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: node_js
 
 node_js:
-  - 6
-  - 7
   - 8
-  - 9
   - 10
 
```

Comment:
test steps should be added here, the PR has already pushed: https://github.com/nkitku/node-oauth2-server/pull/1, please help get it merged